src/main/java/de/ellpeck/actuallyadditions/mod/blocks/BlockLaserRelay.java

```diff
@@ -259,10 +260,10 @@ public void displayHud(Minecraft minecraft, EntityPlayer player, ItemStack stack
 
                         String expl;
                         if(compass){
-                            expl = relay.getCompassDisplayString();
+                            expl = relay.getCompassDisplayString().replaceAll("\\\\n", "\n");
                         }
                         else{
-                            expl = TextFormatting.GRAY.toString()+TextFormatting.ITALIC+"Hold a Compass to modify!";
+                            expl = TextFormatting.GRAY.toString()+TextFormatting.ITALIC+StringUtil.localize("info."+ModUtil.MOD_ID+".laserRelay.mode.noCompasss").replaceAll("\\\\n", "\n");
```

Comment:
Please make it so that everything that is manually cut into two strings is just two strings in the localization file, added together by a manual "\n".
So basically,
`StringUtil.localize(firstPart)+"\n"+StringUtil.localize(secondPart).`